app/routes.py

```diff
@@ -1,2 +1,317 @@
-from flask import Blueprint
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+import os
+import requests
+from datetime import datetime, timedelta
 
+now=datetime.now()
+
+customer_bp = Blueprint("customer", __name__, url_prefix="/customers")
+video_bp = Blueprint("video", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rental", __name__, url_prefix="/rentals")
+
+@video_bp.route("/<video_id>/rentals", methods=["GET"])
+def get_rentals_by_video(video_id):
+
+    rentals = db.session.query(Rental).filter(Rental.video_id ==video_id)
+
+    if rentals is None:
+        return jsonify(None), 404
+
+    elif request.method == "GET":
+        response = []
+
+        for rental in rentals:
+
+            customer = Customer.query.get(rental.customer_id)
+
+            response.append({
+                "due_date": rental.due_date,
+                "name": customer.name,
+                "phone": customer.phone,
+                "postal_code": customer.postal_code
+            })
+
+        return jsonify(response), 200
+
+
+@customer_bp.route("/<customer_id>/rentals", methods=["GET"])
+def get_rentals_by_customer(customer_id):
+
+    rentals = db.session.query(Rental).filter(Rental.customer_id ==customer_id)
+
+    if rentals is None:
+        return jsonify(None), 404
+
+    elif request.method == "GET":
+        response = []
+
+        for rental in rentals:
+
+            video = Video.query.get(rental.video_id)
+
+            response.append({
+                "release_date": video.release_date,
+                "title": video.title,
+                "due_date": rental.due_date
+            })
+
+        return jsonify(response), 200
+
+@rental_bp.route("/check-in", methods=["POST"])
+def return_video():
+    if request.method == "POST":
+
+        request_body = request.get_json()
+
+        if request_body["customer_id"] and request_body["video_id"]:
+            
+            returned_rental = Rental(customer_id=request_body["customer_id"],
+                                video_id=request_body["video_id"])
+
+            customer = Customer.query.get(request_body["customer_id"])
+            video = Video.query.get(request_body["video_id"])
+
+            if not customer or not video:
+                response = "Not found tho"
+                return jsonify(response), 404
+
+            elif customer.videos_checked_out_count != 0:
```

Comment:
Just in case it's somehow negative.

```suggestion
            elif customer.videos_checked_out_count > 0:
```